src/main/com/fulcrologic/rad/attributes.cljc

```diff
@@ -86,10 +90,11 @@
      are defined by reports, forms, database adapters, and rendering plugins. Look for `*-options` namespaces in
      order to find vars with docstrings that describe the possible options."
      [sym & args]
-     (let [[k type m] (if (string? (first args)) (rest args) args)
+     (let [docstring (if (string? (first args)) (first args))
+           [k type m] (if (string? (first args)) (rest args) args)
            nspc (if (comp/cljs? &env) (-> &env :ns :name str) (name (ns-name *ns*)))
            fqkw (keyword (str nspc) (name sym))]
-       `(def ~sym (new-attribute ~k ~type (with-meta ~m {:registration-key ~fqkw}))))))
+       `(def ~sym (new-attribute ~k ~type ~docstring (with-meta ~m {:registration-key ~fqkw}))))))
```

Comment:
This should be a vary-meta on the sym

Reply:
fair enough. See above for context on why I'd like it here too.